lib/Backend/PageAllocatorPool.cpp

```diff
@@ -37,8 +38,10 @@ void PageAllocatorPool::Shutdown()
     AutoCriticalSection autoCS(&cs);
     if (Instance)
     {
-        CloseHandle(Instance->idleCleanupTimer);
-        Instance->RemoveAll();
+        if (Instance->idleCleanupTimer)
+        {
+            CloseHandle(Instance->idleCleanupTimer);
```

Comment:
yes I think, it should not enter here again though. or might be better to keep the data for debugging so we know the timer is initialized correctly